script/test

```diff
@@ -328,6 +327,16 @@ envsetup()
         fi
     fi
 
+    # All expect tests only works in real time mode.
+    VIRTUAL_TIME=1
+    for arg in "$@"; do
+        if [[ $arg == expect ]]; then
+            VIRTUAL_TIME=0
```

Comment:
it's convenient to set VIRTUAL_TIME=0 by default for `expect` tests. But this also bring extra complexity to this `test` script. Maybe just using`VIRTUAL_TIME=0 ./script/test expect` is more consistent?